### PR TITLE
feat(upgrade): Update copy text on upgrade terminal tab

### PR DESCRIPTION
**Pull request by DavidHollins6**
## What kind of change does this PR introduce?

A small copy update

Screenshot 2023-02-06 at 12 15 18

## What steps did you take to test this? This is required before we can merge, make sure to test the flow you've updated.

1. Step A
2. Step B
3. Step C

## Checklist

- [ ] Documentation
- [ ] Testing 
- [ ] Ready to be merged
      
- [ ] Added myself to contributors table

### CODE REVIEW

File: packages/app/src/app/components/Preview/DevTools/Tabs/index.tsx [hunk 1]
```diff
@@ -50,52 +52,54 @@ export const DevToolTabs = ({
   return (
     <Container>
       <Tabs>
-        {panes.map((pane, i) => {
-          const active = !hidden && i === currentPaneIndex;
-          const view = views[pane.id];
+        {panes
+          .filter(pane => (isOnEmbedPage ? !pane.hideOnEmbedPage : true))
```
Comment (DavidHollins6) [inline packages/app/src/app/components/Preview/DevTools/Tabs/index.tsx:56]: This filter is the only new code, everything else is a dodgy git diff

File: packages/app/src/app/components/Preview/DevTools/index.tsx [hunk 2]
```diff
@@ -81,7 +80,6 @@ const VIEWS: IViews = {
   [tests.id]: tests,
   [terminal.id]: terminal,
   [reactDevTools.id]: reactDevTools,
-  [terminalUpgrade.id]: terminalUpgrade,
```
Comment (DavidHollins6) [inline packages/app/src/app/components/Preview/DevTools/index.tsx:84]: This shouldn't be imported here as it is used on the `embed` routes and Overmind is not setup. Instead there is an `addedViews` prop for this purpose.

Comment (alexnm) [reply packages/app/src/app/components/Preview/DevTools/index.tsx:84]: nice!

**Review by alexnm — APPROVED** [review]
Looks good

**Pull request merged**